src/App.tsx

```diff
@@ -6,14 +6,19 @@ import { Product } from './components/Product';
 import { SearchBar } from './components/SearchBar';
 import { TextInput } from './components/TextInput';
 import { lightTheme } from './themes';
+import { Route } from 'react-router-dom';
+import StoreHomePage from './pages/StoreHomePage';
 
 function App() {
   const [theme, setTheme] = useState<DefaultTheme>(lightTheme);
   const { showAlert } = useAlert();
 
   return (
     <ThemeProvider theme={theme}>
-      <CustomButton
+      <Route path="/store" exact={true}>
```

Comment:
```suggestion
      <Route path="/store" exact>
```

여기두 true필요 없어욥 그냥 exact라고 적어두는거 자체가 true가 되어용